pika/channel.py

```diff
@@ -15,7 +15,8 @@
 
 
 LOGGER = logging.getLogger(__name__)
-MAX_CHANNELS = 32768
+
+MAX_CHANNELS = 65535  # per AMQP 0.9.1 spec.
```

Comment:
Prior to this PR, there was a discrepancy between the old value (32768) of this constant and the hard-coded value (65535) used by the old "channels_max" validation logic in `connection.Parameters`.